mobile/src/components/course/components/edit-course/edit-course.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+
+import { AppScreenName } from '~/common/enums/enums';
+import { BackButton, View } from '~/components/common/common';
+import { useAppNavigate, useEffect } from '~/hooks/hooks';
+
+const EditCourse: FC = () => {
+  const navigation = useAppNavigate();
+
+  const navigateToCourseManagement = (): void => {
+    navigation.navigate(AppScreenName.COURSE_MANAGEMENT);
+  };
+
+  useEffect(() => {
+    navigation.setOptions({
+      headerLeft: () => <BackButton onPress={navigateToCourseManagement} />,
+    });
+  }, []);
+
+  return <View />;
```

Comment:
Reuse `EditCourseCategory` here

Reply:
But, pay attention on methods handleSelectNewCategory, navigateToCourseScreen so extract UI component from EditCourseCategory, and then reuse at 2 ones

Reply:
And, after update action(at EditCourse) don`t forget to trigger getCourses one at Courses Screen